hardhat.config.ts

```diff
@@ -1,6 +1,7 @@
 import '@matterlabs/hardhat-zksync-deploy'
 import '@matterlabs/hardhat-zksync-solc'
 import '@matterlabs/hardhat-zksync-verify'
+import '@nomicfoundation/hardhat-toolbox'
```

Comment:
I think it's better to use zksync toolbox(https://github.com/matter-labs/hardhat-zksync/tree/main/packages/hardhat-zksync-toolbox). But generally I see there are imports for the plugins, so why the toolbox is needed?

Reply:
Unfortunately, zksync toolbox and hardhat-toolbox are not same in functionality. Hardhat-toolbox has a wider range of features, such as allowing you to run a project on the ethereum network and conduct debugging, which is currently more difficult to do on the zksync network. Hardhat-toolbox also offers plugins such as solidity-coverage, typechain, and some plugins that make the testing process easier.